test/unit/three.example.unit.js

```diff
@@ -4,3 +4,4 @@ import './example/utils/BufferGeometryUtils.tests';
 import './example/exporters/GLTFExporter.tests';
 import './example/loaders/GLTFLoader.tests';
 import './example/objects/Lensflare.tests';
+import './example/math/Random.tests';
```

Comment:
I've added the Random tests to this list, but I also noticed that the CI configuration doesn't actually run the unit tests for examples. Maybe `npm run test-unit-examples` should be added here? https://github.com/mrdoob/three.js/blob/dev/.github/workflows/ci.yml#L51